test/unit/services/data-test.js

```diff
@@ -106,6 +107,18 @@ describe('InstallerDataService', function() {
     });
   });
 
+  describe('copyUninstaller', function(){
+    it('should be able to handle copyUninstaller', function(){
+      let data = {
+          darwin: ()=>'darwin',
+      };
+      sandbox.stub(Platform,'getOS').returns('darwin');
+      expect(Platform.identify(data)).to.be.equal('darwin');
+      sandbox.spy(svc, 'copyUninstaller');
```

Comment:
After all above is fixed you can fix the issue here https://github.com/redhat-developer-tooling/developer-platform-install/blob/master/browser/services/data.js#L53. Fix is really simple
``` JavaScript
if(Platform.OS === 'win32') {
    this.copyUninstaller();
}
```
This change will fix unit test failure.